### ISSUE TITLE
Jump to "Add ... to ..."

### ISSUE THREAD

**Issue by racke** [interchange/TableEditor#62]
We are creating a list of existing items below the list of related items when we hit the button
"Add existing ....", but we don't jump down to the added list.

This is confusing for the user.

Regards
               Racke

**Comment by gregapompe** [comment]
True, i tried to solve this once, but failed. AngularJS makes this harder than it should be.
